changes/fix_GLT-3920.md

```diff
@@ -0,0 +1 @@
+Fixed bug where an add button would display on the "Edit Assay" page's "Tests" table without being in edit mode.
```

Comment:
Could you update this change message slightly? The `compact-changelogs` script will collate the contents of each `fix_xyz.md` file into its own bullet point under a heading "Fixes" - so, this would look like:

## Fixed
* Fixed bug where an add button would display on...

You can see more examples of this in the [Release notes](https://github.com/miso-lims/miso-lims/blob/develop/RELEASE_NOTES.md#1551---2023-04-06), where we tend to omit the "Fixed bug where" and just describe the bug.